src/test/std-test/ERC4626.test.sol

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity >=0.8.0 <0.9.0;
+
+import "erc4626-tests/ERC4626.test.sol" as StdTest;
+
+/// @dev This is a16z's test, with some extra hooks
+abstract contract ERC4626Test is StdTest.ERC4626Test {
```

Comment:
I wanted to be able to test the end state after each test had run, so I basically copied the tests and added hooks to it